### PR TITLE
I would like to participate - would be great!

### PR DESCRIPTION
**Pull request by yoyogruber**
I would like to register for JS CraftCamp.    
- [x] My Pull Request contains a json file `$firstname_$lastname.json` or `$nickname.json`    
- [x] The json file follows the template at https://github.com/jscraftcamp/website/blob/master/participants/_template.json and contains the mandatory fields `name`, `tags`, `when`, `what_is_my_connection_to_javascript` and `what_can_i_contribute`    
- [x] If I can't attend, I will either send another Pull Request removing my json file or an E-Mail with the subject 'UNREGISTER' to team@jscraftcamp.org

### CODE REVIEW

File: participants/yoyo_gruber.json [hunk 1]
```diff
@@ -1,14 +1,17 @@
 {
   "name": "Hans-Georg Gruber",
-  "company": "self",
-  "tags": ["Marketing_Automation", "FormValidation", ],
+  "company": "Demand Systems GmbH",
+  "tags": ["Marketing_Automation", "FormValidation"],
   "when": {
+    "friday": true,
+    "friday_party": true,
     "saturday": true,
```
Comment (choas) [inline participants/yoyo_gruber.json]: comma causes error

File: participants/yoyo_gruber.json [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-    "sunday": true,
   },
+  "email": "",
+  "dietary_requirements": "",
   "what_is_my_connection_to_javascript": "user",
   "what_can_i_contribute": "javascript on forms, javascript for Marketing Automation, Form validation",
-  "tshirt": "",
+  "tshirt": "/^(M)-(M)$/",
```
Comment (choas) [inline participants/yoyo_gruber.json]: should me "M-M"

File: participants/yoyo_gruber.json [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
   "urls": {
```
Comment (choas) [inline participants/yoyo_gruber.json]: all urls need content (see result of 'npm test')

**Comment by davelosert** [comment]
@yoyogruber : Maybe you want to share a little bit more about your connection with JavaScript? Would be nice to hear what actually motivates you to participate in a JSCraftCamp :) . Have a look what other Participants wrote: http://jscraftcamp.org/participants.html

You want to give us some more information and update your PR / Registration?

**Comment by yoyogruber** [comment]
added more motivation and areas of interest and more possible contribution topics.

**Comment by davelosert** [comment]
@yoyogruber : Great, thank you so much for taking the time! Marketing Automation is for sure an interesting topic for a lot of people joining our event.
I'm looking forward on seeing you at JSCC17!

**Pull request merged**